docs/source/conf.py

```diff
@@ -128,6 +128,7 @@
     r"https://docs.qualcomm.com/*",
     # TODO(jambayk): remove this when the issue is fixed
     r"https://www.intel.com/*",
+    # TODO(team): html files are generated after doc build. Linkcheck doesn't work for them.
+    # Remove this when linkcheck works for html files.
     r".*\.html",
```

Comment:
could this be narrowed down to links that don't with https so that we can still ensure other links are correct?

Reply:
Sounds good to me!